lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -84,6 +84,12 @@ def self.available_options
                                        verify_block: proc do |value|
                                          UI.user_error!("firebase_app_distribution: Couldn't find ipa file at path '#{value}'") unless File.exist?(value)
                                        end),
+          FastlaneCore::ConfigItem.new(key: :gsp_path,
+                                       env_name: "GOOGLESERVICE_PLIST_PATH",
```

Comment:
Consider GOOGLESERVICE_INFO_PLIST_PATH, based on my (very brief googling) I think this file is more commonly referred to as GoogleService-Info (_with_  the info).